strawberry/experimental/pydantic/object_type.py

```diff
@@ -144,6 +144,7 @@ def wrap(cls):
         )
 
         model._strawberry_type = cls  # type: ignore
+        cls._pydantic_type = model  # type: ignore
```

Comment:
this is a patter we want to move away from :)

I think we can keep this for now though, we still need to do some refactoring anyway ^^